.goreleaser.yml

```diff
@@ -162,6 +162,14 @@ brews:
       bin.install "ttn-lw-cli"
       libexec.install doc
 
+s3:
+  - bucket: {{ or (index .Env "AWS_BUCKET") "the-things-stack-releases" }}
+    ids:
+    - stack
+    - cli
+    region: {{ or (index .Env "AWS_REGION") "eu-west-1" }}
+    folder: "{{ .Version }}"
```

Comment:
Does it create the folder if it doesn't exist?

Reply:
There is not actually such thing as a folder in s3. It's just a key/filename that contains slashes.